.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug Report 
+about: Report an issue
+---
+## Prerequisites
+<!--- Go through the items below before logging an issue -->
+- [ ] Please search this repository's issues to see if it has already been reported. This helps reduce duplicate issues from being created.
+- [ ] If this is a Security Disclosure please follow the guidelines in [CONTRIBUTING](../CONTRIBUTING.md). This helps keep folks from accidentally releasing vulnerabilities before the maintainers get a chance to fix the issue.

```

Comment:
Let's replace `[CONTRIBUTING](../CONTRIBUTING.md)` with `[Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy)`

Reply:
I changed it pointing to the SECURITY.md file in the root folder